content/docs/csidriver/installation/operator/operator_migration.md

```diff
@@ -0,0 +1,73 @@
+---
+title: CSI to CSM Operator Migration
+description: >
+  Migrating from CSI Operator to CSM Operator
+---
+
+## CR Sample Files
+
+{{<table "table table-striped table-bordered table-sm">}}
+|            |    CSI Operator     |    CSM Operator     |
+|------------|:----------:|:----------:|
+| PowerScale |    [isilon_v270_k8s_127.yaml](https://github.com/dell/dell-csi-operator/blob/main/samples/isilon_v270_k8s_127.yaml)    |     [storage_csm_powerscale_v280.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_powerscale_v280.yaml)       |
+| PowerMax   |    [powermax_v270_k8s_127.yaml](https://github.com/dell/dell-csi-operator/blob/main/samples/powermax_v270_k8s_127.yaml)    |     [storage_csm_powermax_v280.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_powermax_v280.yaml)       |
+| PowerStore |    [powerstore_v270_k8s_127.yaml](https://github.com/dell/dell-csi-operator/blob/main/samples/powerstore_v270_k8s_127.yaml)    |     [storage_csm_powerstore_v290.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_powerstore_v290.yaml)       |
+| Unity XT      |    [unity_v270_k8s_127.yaml](https://github.com/dell/dell-csi-operator/blob/main/samples/unity_v270_k8s_127.yaml)    |     [storage_csm_unity_v280.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_unity_v280.yaml)       |
+| PowerFlex  |     [vxflex_v270_k8s_127.yaml](https://github.com/dell/dell-csi-operator/blob/main/samples/vxflex_v270_k8s_127.yaml)       |      [storage_csm_powerflex_v290.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_powerflex_v290.yaml)      |
+{{</table>}}
+>NOTE: Sample files refer to the latest version for each platform. If you do not want to upgrade, please find your preferred version in the linked [csm-operator repo](https://github.com/dell/csm-operator/blob/main/samples).
+
+## Migration Steps
+
+1. Save the CR yaml file of the current CSI driver to preserve the settings. Use the following commands in your cluster to get the CR:
+  ```
+      kubectl -n <namespace> get <CRD_kind>
+      kubectl -n <namespace> get <CRD_kind>/<CR_name> -o yaml
+  ```
+  Example for CSI Unity:
+  ```
+      kubectl -n openshift-operators get CSIUnity
+      kubectl -n openshift-operators get CSIUnity/test-unity -o yaml
+  ```
+2. Map and update the settings from the CR in step 1 to the relevant CSM Operator CR
```

Comment:
It would be nice to give an example for this. This step seems a bit unclear to me (although I understand what is the reason).